### PR TITLE
Playground form

### PR DESCRIPTION
**Pull request by Tymek**
## About the changes
Adds top part of UI for playground feature - for selecting environment and projects, and setting context.

### Important files
`src/component/playground/Playground/Playground.tsx`

### CODE REVIEW

File: src/component/playground/Playground/Playground.tsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+                <Box component="form" onSubmit={onSubmit}>
+                    <Typography
+                        sx={{
+                            mb: 3,
+                        }}
+                    >
+                        Configure playground
+                    </Typography>
+                    <PlaygroundConnectionFieldset />
+                    <Divider
```
Comment (FredrikOseberg) [inline src/component/playground/Playground/Playground.tsx:50]: Could this be a `StyledDashedDivider`, then we could reuse it later and we wouldn't have to repeat the styles beneath. Might clean up the render function a bit.

File: src/component/playground/Playground/PlaygroundCodeFieldset/PlaygroundCodeFieldset.tsx [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+                    {
+                        ...currentValue,
+                        [contextField]: contextValue,
+                    },
+                    null,
+                    2
+                )
+            );
+        } catch (error) {
+            // Shouldn't happened, because button is disabled when there is an error
```
Comment (FredrikOseberg) [inline src/component/playground/Playground/PlaygroundCodeFieldset/PlaygroundCodeFieldset.tsx:81]: So if I understand this correctly, there will be a delay where the button will be functional before the error is updated. Should we give any feedback if that is the case or is the button being disabled enough feedback?

**Review by FredrikOseberg — CHANGES_REQUESTED** [review]

File: src/component/playground/Playground/Playground.tsx (hunk 1 above)
Comment (Tymek) [reply src/component/playground/Playground/Playground.tsx:50]: I'll add a variant to theme.

File: src/component/playground/Playground/Playground.tsx [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    const [context, setContext] = useState<string>();
+    const [contextObject, setContextObject] = useState<string>();
+    const onSubmit: FormEventHandler<HTMLFormElement> = event => {
+        event.preventDefault();
+
+        try {
+            setContextObject(
+                JSON.stringify(JSON.parse(context || '{}'), null, 2)
+            );
+        } catch (e) {}
```
[the remark this answers is not in the document]
Comment (Tymek) [reply src/component/playground/Playground/Playground.tsx]: toast message added

File: src/component/playground/Playground/Playground.tsx (hunk 1 above)
Comment (Tymek) [reply src/component/playground/Playground/Playground.tsx:59]: First divider is dashed, the second one isn't

File: src/component/playground/Playground/PlaygroundCodeFieldset/PlaygroundCodeFieldset.tsx [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+}
+
+export const PlaygroundCodeFieldset: VFC<IPlaygroundCodeFieldsetProps> = ({
+    value,
+    setValue,
+}) => {
+    const theme = useTheme();
+    const { setToastData } = useToast();
+    const { context } = useUnleashContext();
+    const contextOptions = useMemo(
```
Comment (FredrikOseberg) [inline src/component/playground/Playground/PlaygroundCodeFieldset/PlaygroundCodeFieldset.tsx:37]: Is this resolved?

**Review by FredrikOseberg — CHANGES_REQUESTED** [review]
Just need a clarification here, but this should be good to go afterwards.

**Review by FredrikOseberg — APPROVED** [review]
LGTM!

**Pull request merged**